cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -163,7 +181,9 @@ def index_item(item, skip_index=False):
                 item_index['id'] = item_id
                 if item.start:
                     item_index['start_date'] = item.start
```

Comment:
not your doing but this can be placed in one line:

`item_index['start_date'] = item.start if item.start`